Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/distance.h

```diff
@@ -1310,8 +1322,1295 @@ double approximate_symmetric_Hausdorff_distance(const TriangleMesh& tm1,
+  const auto vpm1 = parameters::choose_parameter(
+    parameters::get_parameter(np1, internal_np::vertex_point),
+    get_const_property_map(vertex_point, tm1));
+  const auto vpm2 = parameters::choose_parameter(
+    parameters::get_parameter(np2, internal_np::vertex_point),
+    get_const_property_map(vertex_point, tm2));
+
+  const bool match_faces1 = parameters::choose_parameter(
+    parameters::get_parameter(np1, internal_np::match_faces), true);
+  const bool match_faces2 = parameters::choose_parameter(
+    parameters::get_parameter(np2, internal_np::match_faces), true);
+  const bool match_faces = match_faces1 && match_faces2;
+
+  // TODO: should we return a union of these realizing triangles?
+  auto out1 = parameters::choose_parameter(
+    parameters::get_parameter(np1, internal_np::output_iterator),
+    CGAL::Emptyset_iterator());
+  auto out2 = parameters::choose_parameter(
+    parameters::get_parameter(np2, internal_np::output_iterator),
+    CGAL::Emptyset_iterator());
+
+  CGAL_precondition(error_bound >= 0.0);
+  const FT error_threshold = static_cast<FT>(error_bound);
+  return internal::bounded_error_symmetric_Hausdorff_impl<Concurrency_tag, Traits>(
+    tm1, tm2, error_threshold, -FT(1), match_faces, vpm1, vpm2, np1, np2, out1, out2);
+}
+
+template< class Concurrency_tag,
+          class TriangleMesh1,
+          class TriangleMesh2,
+          class NamedParameters1 >
+double bounded_error_symmetric_Hausdorff_distance(
+  const TriangleMesh1& tm1,
+  const TriangleMesh2& tm2,
+  const double error_bound,
+  const NamedParameters1& np1)
+{
+  return bounded_error_symmetric_Hausdorff_distance<Concurrency_tag>(
+    tm1, tm2, error_bound, np1, parameters::all_default());
+}
+
+template< class Concurrency_tag,
+          class TriangleMesh1,
+          class TriangleMesh2>
+double bounded_error_symmetric_Hausdorff_distance(
+  const TriangleMesh1& tm1,
+  const TriangleMesh2& tm2,
+  const double error_bound = 0.0001)
+{
+  return bounded_error_symmetric_Hausdorff_distance<Concurrency_tag>(
+    tm1, tm2, error_bound, parameters::all_default());
 }
-} // end of namespace CGAL::Polygon_mesh_processing
+
+// TODO: Find better name!
+// TODO: Should we use one-sided or symmetric distance here?
+
+/**
+ * \ingroup PMP_distance_grp
+ * returns `true` if the Hausdorff distance between two meshes is larger than
+ * the user-defined max distance, otherwise it returns `false`. The distance used
+ * to compute the proximity of the meshes is the `bounded_error_Hausdorff_distance`.
+ *
+ * Instead of computing the full distance and checking it against the user-provided
+ * value, this function early quits in case certain criteria show that the meshes
+ * do not satisfy the provided distance bound.
+ *
+ * @return Boolean `true` or `false`
+ */
+template< class Concurrency_tag,
+          class TriangleMesh1,
+          class TriangleMesh2,
+          class NamedParameters1,
+          class NamedParameters2 >
+bool is_further_than(
```

Comment:
`is_Hausdorff_distance_larger`?

Reply:
cc @CGAL/geometryfactory

Reply:
I changed it since it sounds better than my name.

Reply:
What about something with `compare` in the name?

Reply:
In fact what is not good about the name is that you offer several Hausdorff distances.   Do you want this function for a particular one, or would it make sense for all of them?  If for all why the particular one? And if a particular one its name should reflect the choice.

Reply:
Several distances?

Reply:
approximate, symmetric....

Reply:
Dmitry just explained that the predicate  only makes sense for the bounded version, But you still have to clarify if bounded_symmetric is also an option.

Reply:
I have checked and, in fact, inside the function that is called now `is_Hausdorff_distance_larger`, I have an option to choose between using one-sided and symmetric bounded-error Hausdorff distance implementation. So, we have two options:

1. We can add this option to the named parameters and put one-sided distance as default.
2. We can create two functions: `is_bounded_error_Hausdorff_distance_larger` and `is_bounded_error_symmetric_Hausdorff_distance_larger`.

I would go for the named parameter because it is cleaner and does not create a bunch of functions with similar behavior.

Reply:
ok for for the named parameter.